docs/docs/user-guide/using-cli.md

```diff
@@ -91,6 +91,20 @@ You can use the `@` token in any command, with any option that accepts a value.
 m365 spo sitescript add --title "Contoso" --description "Contoso theme script" --content `@themeScript.json
 ```
 
+## @meId and @meUsername tokens
```

Comment:
Let's elaborate a bit where this information is coming from and what are the values to which the tokens resolve